### ISSUE TITLE
I cannot send/receive chat messages from the global chat

### ISSUE THREAD

**Issue by RobChiocchio** [phillmac/IRC-to-TS3-chat#3]
I do not see any options in the configuration file (or the code) to use the global server chat room on TeamSpeak instead of picking a specific channel. Is this/will this be an option?
-Rob

**Comment by phillmac** [comment]
Yes this seems to be technically possible, receiving and sending server channel text messages is supported by the api. I'll look into how hard it would be to add some options to enable this.

**Comment by Bluscream** [comment]
BluscreamFanBoy/multistreamchat

**Comment by phillmac** [comment]
@Bluscream Your app doesn't even seem to support teamspeak, so please don't go advertising it here.

**Comment by Bluscream** [comment]
Its neither my app, nor isn't it supporting TS. I hoped it can help you with that.
